test/test_jit.py

```diff
@@ -143,6 +144,10 @@
 import warnings
 import zipfile
 
+# This assertion fails when run in internal CI
+if not IS_SANDCASTLE and not IS_FBCODE:
```

Comment:
@mruberry, just making sure, this is what we want, right?

Reply:
Yep